search/src/main/nodejs/search_create_default_channel.js

```diff
@@ -54,19 +58,29 @@ async function createDefaultChannel(searchData) {
 
   // if not a png, transform to png
   if (/\.(tiff?|gif|jpe?g|bmp)$/.test(upload)) {
-    console.log(`Converting uploaded image to png`);
-    const pngExt = ".png";
-    const image = await Jimp.read(imageContent);
-    const imageBuffer = await image.getBufferAsync(pngMime);
-    const pngImageName = getSearchKey(fullSearchInputImage, pngExt);
-    sourceImage = pngImageName;
-    channelName = upload.replace(/\.([^.]*)$/, "_1.png");
-    await putS3Content(searchBucket, pngImageName, pngMime, imageBuffer);
-    searchMetaData.displayableMask = getSearchMaskId(pngImageName, pngExt);
-    console.log(`Image conversion complete`);
+    try {
```

Comment:
this branch should now be tested by methods from search_create_default_channel.test.js